app/lib/application_extension.rb

```diff
@@ -40,6 +40,8 @@ def close_streaming_sessions(resource_owner = nil)
           pipeline.publish("timeline:access_token:#{id}", payload)
         end
       end
+
+      redis.publish('system', Oj.dump(event: :terminate, access_tokens: tokens.ids))
```

Comment:
Here, if an application had say, 500 access tokens before being deleted, we'd previously publish 500 events in one redis pipeline to 500 pubsub topics.

With this change, we'd publish only one event with 500 access token IDs in the message body to a single pubsub topic.

This would apply for each multiple of 1000, so for 1500 access tokens, we'd send two publishes to a single topic.